README.md

```diff
@@ -2,6 +2,17 @@
 
 beaker-rspec is a bridge between the puppet acceptance test harness ([beaker](https://github.com/puppetlabs/beaker)) and [rspec](https://github.com/rspec/rspec). It also integrates [serverspec](http://serverspec.org/).
 
+#Upgrading from beaker-rspec 5 to 6
+
+In beaker-rspec 6, we've picked up the newest beaker, 3.y. In this release, we've
+given up support for Ruby 1.9 and moved to 2.2.5 as our lowest tested version,
```

Comment:
If we're giving up 1.9 can we delete the special-case code in the gemspec too?

* https://github.com/puppetlabs/beaker-rspec/pull/87/files#diff-1aa0c7f074a678ea98900da6c3caf004R7
* https://github.com/puppetlabs/beaker-rspec/pull/87/files#diff-1aa0c7f074a678ea98900da6c3caf004R30
* https://github.com/puppetlabs/beaker-rspec/pull/87/files#diff-1aa0c7f074a678ea98900da6c3caf004R34